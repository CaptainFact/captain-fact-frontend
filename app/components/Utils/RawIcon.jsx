import React from 'react'


export const RawIcon = ({name}) =>
  <i className={`fa icon-${name}`}/>

export default RawIcon