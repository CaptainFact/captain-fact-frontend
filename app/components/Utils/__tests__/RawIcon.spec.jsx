import RawIcon from '../RawIcon'

test('render icon', () => {
  snapshot(<RawIcon name="plus"/>)
})
