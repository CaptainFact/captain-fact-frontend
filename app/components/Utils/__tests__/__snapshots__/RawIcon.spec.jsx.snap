// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render icon 1`] = `
<i
  className="fa icon-plus"
/>
`;
